packages/design-system/src/components/web-components/ds-autocomplete/ds-autocomplete.stories.tsx

```diff
@@ -0,0 +1,284 @@
+import { useEffect, useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import type { Meta } from '@storybook/react';
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { webComponentDecorator } from '../storybook';
+import './ds-autocomplete';
+import { AutocompleteItem } from '../../Autocomplete';
+import uniqueId from 'lodash/uniqueId';
+import '../ds-text-field';
+
+const meta: Meta = {
+  title: 'Web Components/ds-autocomplete',
+  decorators: [webComponentDecorator],
+  parameters: {
+    docs: {
+      page: WebComponentDocTemplate,
+    },
+  },
+  args: {
+    'clear-input-text': 'Clear search',
+    'clear-search-button': true,
+    'loading-message': 'Loading...',
+    'no-results-message': 'No results',
+    name: 'ds-autocomplete',
+  },
+  argTypes: {
```

Comment:
I think you might want to add this property ```table: {
        disable: true,
      },``` to the following items: textFieldLabel, textFieldHint and children so they don't appear in the attributes table.

Reply:
Thanks!